ceph-docker-pull-requests/build/build

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -ex
+
+sudo apt-get install -y --force-yes docker
+docker --version
+sudo apt-get install -y --force-yes xfsprogs
+sudo "$WORKSPACE"/travis-builds/purge_cluster.sh
+sudo "$WORKSPACE"/travis-builds/build_imgs.sh
+
+# NOTE(leseb): somehow putting everything in a 'script' task does not work
+# so we have to split it up that way.
+# It seems that we have an issue when not running 'docker run' from an install step
+#install:
+sudo "$WORKSPACE"/travis-builds/prepare_osd_fs.sh
+sudo docker run -d --name ceph-mon --net=host -v /etc/ceph:/etc/ceph -v /var/lib/ceph:/var/lib/ceph -e MON_IP=$(ip -4 -o a | awk '/eth|ens|eno|enp/ { sub ("/..", "", $4); print $4 }') -e CEPH_PUBLIC_NETWORK=$(grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}/[0-9]\{1,2\}' /proc/net/fib_trie | grep -vE "^127|0$" | head -1) daemon mon
```

Comment:
all of this is mostly a literal port of the travis config, I didn't want to deviate at all until we get something working